### PR TITLE
influxdb@1 1.8.4 (new formula)

### PR DESCRIPTION
**Pull request by denispershin**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Reason:
Many people use 1.8.4 version and don't want upgrade to 2.x

### CODE REVIEW

File: Formula/influxdb@1.8.rb [hunk 1]
```diff
@@ -0,0 +1,101 @@
+class InfluxdbAT18 < Formula
+  desc "Time series, events, and metrics database"
+  homepage "https://influxdata.com/time-series-platform/influxdb/"
+  url "https://github.com/influxdata/influxdb/archive/v1.8.4.tar.gz"
+  sha256 "9f2c135c8f9f50ca469196e6b4e575e26f1a338538788e71b664212e03b4df7b"
+  license "MIT"
+  head "https://github.com/influxdata/influxdb.git"
```
Comment (carlocab) [inline Formula/influxdb@1.8.rb:7]:
```suggestion
```

File: Formula/influxdb@1.8.rb [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+  # The regex below omits a rogue `v9.9.9` tag that breaks version comparison.
+  livecheck do
+    url :stable
+    regex(/^v?((?!9\.9\.9)\d+(?:\.\d+)+)$/i)
```
Comment (carlocab) [inline Formula/influxdb@1.8.rb:12]:
```suggestion
    regex(/^v?((?!9\.9\.9)1\.(?:\.\d+)+)$/i)
```

File: Formula/influxdb@1.8.rb [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  end
+
+  bottle do
+    sha256 cellar: :any_skip_relocation, arm64_big_sur: "0a14b0bb22f27de27b072abf29ab014cc9723e2871904700471f78e6d9f6f030"
+    sha256 cellar: :any_skip_relocation, big_sur:       "09ed45e33953ab84c6f5b8a862da62135164a10926c81186ba5587a7f65a0041"
+    sha256 cellar: :any_skip_relocation, catalina:      "6d88fe5e33124a220eb70d8e140fdb213b13d465560ef04a8bc3a9b992d1dd94"
+    sha256 cellar: :any_skip_relocation, mojave:        "ad332849b3ab8d6a07e58a06a2164ccc207652797f55e396f3845081a006a121"
+  end
```
Comment (carlocab) [inline Formula/influxdb@1.8.rb:20]:
```suggestion
  keg_only :versioned_formula
```

**Review by carlocab — CHANGES_REQUESTED** [review]
Thanks for your PR, @denispershin.

Does influxdb 1.8 meet the [requirements for versioned formulae](https://docs.brew.sh/Versions#acceptable-versioned-formulae)? In particular, is version 1.8 still under active development?

Also, please use the preferred commit-message style for homebrew/core. We put the name of the formula first in commit-message headings. Refer to the [documentation](https://docs.brew.sh/Formula-Cookbook#commit) for more details.

**Comment by carlocab** [comment]
I cleaned this up a bit for you.

File: Formula/influxdb@1.rb [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+  desc "Time series, events, and metrics database"
+  homepage "https://influxdata.com/time-series-platform/influxdb/"
+  url "https://github.com/influxdata/influxdb/archive/v1.8.4.tar.gz"
+  sha256 "9f2c135c8f9f50ca469196e6b4e575e26f1a338538788e71b664212e03b4df7b"
+  license "MIT"
+
+  # The regex below omits a rogue `v9.9.9` tag that breaks version comparison.
+  livecheck do
+    url :stable
+    regex(/^v?((?!9\.9\.9)1.+(?:\.\d+)+)$/i)
```
Comment (carlocab) [inline Formula/influxdb@1.rb:11]:
```suggestion
    regex(/^v?(1(?:\.\d+)+)$/i)
```
Sorry, messed up that regex. This one should work. Squash your commits together when you make this change, please. Thanks!

Comment (denispershin) [reply Formula/influxdb@1.rb]: Thank you very much for help with this MR!

**Review by carlocab — DISMISSED** [review]
Thanks for your first contribution to homebrew/core, @denispershin.

**Review by carlocab — APPROVED** [review]
Missed something else. This should be fixed now, I think. 🤞

**Comment by carlocab** [comment]
Just for future reference: there is a [release candidate for 1.8.5](https://github.com/influxdata/influxdb/releases/tag/v1.8.5rc0), so it looks like 1.8.x is still under active development.

**Pull request closed without merging**